From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Hauke Mehrtens <hauke@hauke-m.de>
Date: Sat, 23 Mar 2024 19:11:15 +0100
Subject: packet_capture: Fix compilation with GCC 13

Fix the following compile problem with GCC 13:
src/network_inspectors/packet_capture/packet_capture.h:25:54: error: 'int16_t' does not name a type
   25 | void packet_capture_enable(const std::string&, const int16_t g = -1, const std::string& t = "");
---
 src/network_inspectors/packet_capture/packet_capture.h | 1 +
 1 file changed, 1 insertion(+)

--- a/src/network_inspectors/packet_capture/packet_capture.h
+++ b/src/network_inspectors/packet_capture/packet_capture.h
@@ -22,6 +22,7 @@
 
 #include <cstdint>
 #include <string>
+#include <cstdint>
 
 void packet_capture_enable(const std::string&, const int16_t g = -1, const std::string& t = "");
 void packet_capture_disable();
